.github/workflows/release.yml

```
@@ -0,0 +1,104 @@
+name: Auto Release
+
+on:
+  push:
+    branches: [ main ]
+    paths: [ 'manifest.json' ]
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)